### PR TITLE
Remove Android user authorizer

### PR DESCRIPTION
**Pull request by kaidokert**
b/352168728

### CODE REVIEW

File: starboard/android/apk/app/src/main/java/dev/cobalt/account/AccessToken.java [hunk 1]
```diff
@@ -1,43 +0,0 @@
...
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package dev.cobalt.account;
-
-import dev.cobalt.util.UsedByNative;
-
-/** POJO holding the token for a signed-in user. */
-public class AccessToken {
```
Comment (zhongqiliang) [inline starboard/android/apk/app/src/main/java/dev/cobalt/account/AccessToken.java:20]: this file can not be deleted. we have reference of it here https://source.corp.google.com/piper///depot/google3/java/com/google/android/libraries/youtube/tv/services/accountmanager/AccountManager.java;l=260?q=dev.cobalt.account.AccessToken
we need to move it to google3 before it delete it.

**Review by zhongqiliang — APPROVED** [review]
LGTM, please do not submit yet. We need some coordination with submit on google3.

File: starboard/android/apk/app/src/main/java/dev/cobalt/account/AccessToken.java (hunk 1 above)
Comment (kaidokert) [reply starboard/android/apk/app/src/main/java/dev/cobalt/account/AccessToken.java:20]: Ack, i'll back this out

Comment (zhongqiliang) [reply starboard/android/apk/app/src/main/java/dev/cobalt/account/AccessToken.java]: you do not have to.

File: starboard/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java [hunk 2]
```diff
@@ -665,12 +653,9 @@ AudioPermissionRequester getAudioPermissionRequester() {
     return audioPermissionRequester;
   }
 
-  void onActivityResult(int requestCode, int resultCode, Intent data) {
-    userAuthorizer.onActivityResult(requestCode, resultCode, data);
-  }
+  void onActivityResult(int requestCode, int resultCode, Intent data) {}
```
Comment (zhongqiliang) [inline starboard/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java:656]: this should be removed, as well as CobaltActivity.onActivityResult()

**Comment by kaidokert** [comment]
Leaving to you @zhongqiliang to land safely here https://github.com/youtube/cobalt/pull/3818

**Pull request closed without merging**